### PR TITLE
Create scalability tests

### PR DESCRIPTION
**Pull request by DanNiESh**
Closes: https://github.com/nerc-project/operations/issues/319

### CODE REVIEW

**Comment by joachimweyl** [comment]
@DanNiESh is this ready for review? If so please request reviewers.

File: scalability_test/create_notebook.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+num_notebooks=$1
+# Number of requests to make concurrently
+batch_size=$2
+username=$3
+SECONDS=0
+
+create_resources() {
+    notebook_name=$1
+    # Create service account
+    sed "s/ope-placeholder/$notebook_name/g" sa_template.yaml | oc apply -f -
```
Comment (larsks) [inline scalability_test/create_notebook.sh:18]: It doesn't appear to be necessary to create the serviceaccount -- this is created automatically in response to creating the notebook (and will be deleted automatically when the notebook is deleted).

File: scalability_test/create_notebook.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    # Create pvc
+    sed "s/ope-placeholder/$notebook_name/g" pvc_template.yaml | oc apply -f -
+
+    # Create notebook
+    sed -e "s/ope-placeholder/$notebook_name/g" -e "s/your_username/$username/g" notebook_template.yaml | oc apply -f -
+}
+
+for ((i=0; i<$num_notebooks; i+=batch_size)); do
+    for ((j=0; j<$batch_size && (i+j)<$num_notebooks; j++)); do
+        notebook_name="ope-scalability-$((i+j))"
```
Comment (larsks) [inline scalability_test/create_notebook.sh:29]: These resource names won't be unique between different test runs, which will result in conflicts if you start a new test before cleaning up an old test, or if two people run tests at the same time. We should include something like a random hash in all names.

File: scalability_test/pvc_template.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ope-placeholder
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4Gi
```
Comment (larsks) [inline scalability_test/pvc_template.yaml:10]: Can this be smaller? Should it be configurable?

**Review by larsks — CHANGES_REQUESTED** [review]

**Comment by larsks** [comment]
@DanNiESh I'd like to suggest https://github.com/OCP-on-NERC/ope-tests/pull/2 as an alternative that implements the suggestions I've made on this PR. Let me know what you think.

File: scalability_test/create_notebook.sh (hunk 1 above)
Comment (DanNiESh) [reply scalability_test/create_notebook.sh:18]: You are right. I double checked that service account is created/deleted automatically.

File: scalability_test/pvc_template.yaml (hunk 3 above)
Comment (DanNiESh) [reply scalability_test/pvc_template.yaml:10]: Yes, it could be as small as 1Gi.

**Comment by DanNiESh** [comment]
I can't approve it because I'm the owner of this PR. @IsaiahStapleton @dystewart Do you want to review it?

**Review by larsks — APPROVED** [review]

**Pull request merged**